suse_migration_services/prechecks/sshd.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2025 SUSE LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>#
+import logging
+import re
+
+from suse_migration_services.defaults import Defaults
+from suse_migration_services.command import Command
+
+log = logging.getLogger(Defaults.get_migration_log_name())
+
+
+def root_login(migration_system=False):
+    """
+    Check whether sshd is enabled and root login is allowed. If so,
+    warn user that they might want to enable it back before starting
+    the migration.
+    """
+    system_root = Defaults.get_system_root_path() if migration_system else '/'
+    sshd_enabled = Command.run(
+        [
+            'systemctl',
+            '--root', system_root,
+            '--quiet',
+            'is-enabled', 'sshd'
+        ]
+    ).returncode == 0
+    if not sshd_enabled:
+        return
+
+    # TODO: check whether root login is disabled
+    sshd_t_output = Command.run(['sshd', '-T']).output.lower()
+    match = re.search('^permitrootlogin (.*)$', sshd_t_output, re.MULTILINE)
+    permit_root_login = match.group(1).strip()
+    if permit_root_login == 'no':
+        # root login already disabled
+        return
+
+    log.warning(
+        """Root login by ssh will be disabled by default in SLE16.
+You might want to enable it back before starting the migration.
+To do so, create /etc/ssh/sshd_config.d/50-permit-root-login.conf
+with the following content:
+
+PermitRootLogin yes
+
+Once the migration is complete, you can install
+openssh-server-config-rootlogin package and remove the previously
+added file."""
+    )
```

Comment:
Can we please use 

```python3
from textwrap import dedent

message = dedent('''\n
    your long text
    and more long text
''')

log.warning(message)
```

It is so much better to read compared to the inline text messages. Thanks